xml/admin_ceph_gateway.xml

```diff
@@ -2692,349 +2689,281 @@ rgw_override_bucket_index_max_shards = 12
-     creation itself because it makes scripting and use of configuration
-     management tools easier later on.
-    </para>
-   </tip>
-   <para>
-    For the purpose of this example, let us assume that the access and secret
-    keys are set in the environment variables:
-   </para>
-<screen># SYSTEM_ACCESS_KEY=1555b35654ad1656d805
-# SYSTEM_SECRET_KEY=h7GhxuBLTrlhVUyxSPUKUV8r/2EI4ngqJxD7iBdBYLhwluN30JaT3Q==</screen>
-   <para>
-    Generally, access keys consist of 20 alphanumeric characters, while secret
-    keys consist of 40 alphanumeric characters (they can contain +/= characters
-    as well). You can generate these keys in the command line:
-   </para>
-<screen># SYSTEM_ACCESS_KEY=$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 20 | head -n 1)
-# SYSTEM_SECRET_KEY=$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 40 | head -n 1)</screen>
-  </sect2>
-
-  <sect2 xml:id="ceph-rgw-fed-naming">
-   <title>Naming Conventions</title>
-   <para>
-    This example describes the process of setting up a master zone. We will
-    assume a zonegroup called <literal>us</literal> spanning the United States,
-    which will be our master zonegroup. This will contain two zones written in
-    a <replaceable>ZONEGROUP</replaceable>-<replaceable>ZONE</replaceable>
-    format. This is our convention only and you can choose a format that you
-    prefer. In summary:
-   </para>
-   <itemizedlist>
-    <listitem>
-     <para>
-      Master zonegroup: United States <literal>us</literal>
-     </para>
-    </listitem>
-    <listitem>
-     <para>
-      Master zone: United States, East Region 1: <literal>us-east-1</literal>
-     </para>
-    </listitem>
-    <listitem>
-     <para>
-      Secondary zone: United States, East Region 2:
-      <literal>us-east-2</literal>
-     </para>
-    </listitem>
-    <listitem>
-     <para>
-      Secondary zone: United States, West Region: <literal>us-west</literal>
-     </para>
-    </listitem>
-   </itemizedlist>
+  <sect2 xml:id="ceph-rgw-multi-req-assump">
+   <title>Requirements and Assumptions</title>
    <para>
-    This will be a part of a larger realm named <literal>gold</literal>. The
-    <literal>us-east-1</literal> and <literal>us-east-2</literal> zones are
-    part of the same &ceph; cluster, <literal>us-east-1</literal> being the
-    primary one. <literal>us-west</literal> is in a different &ceph; cluster.
+     A multi-site configuration requires at least two &ceph; storage clusters, and
+     at least two &cogw; instances, one for each &ceph; storage cluster.
+     The following configuration assumes at least two &ceph; storage clusters
+     are in geographically separate locations. However, the configuration can
+     work on the same site. For example, named <literal>rgw1</literal> and
+     <literal>rgw2</literal>.
    </para>
+   <para>A multi-site configuration requires a master &zonegroup; and a
+     master zone. Additionally, each &zonegroup; requires a master zone.
```

Comment:
Maybe we introduce a one liner about master zones/zonegroups, we do mention this more than a few times, but don't define what a master zone is. 
A master zone is the source of truth with respect to all metadata operations in a multisite cluster.